packages/form/src/CheckboxGroup.stories.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import { StoryObj } from '@storybook/react';
+import { Button } from 'reactstrap';
+import * as yup from 'yup';
+import '@availity/date/styles.scss';
+
+import { Checkbox, CheckboxGroup } from '.';
+// import README from '../form/README.md';
+
+import FormResults from '../../../story-utils/FormikResults';
+
+export default {
+  title: 'Form Components/Form/CheckboxGroup',
+  component: CheckboxGroup,
+  parameters: {
+    docs: {
+      // page: README,
+      description: {
+        component: 'Availity form component that is wired to be hooked up to formik.',
+      },
+    },
+  },
+  args: {
+    required: false,
+  },
+};
+
+type FormStoryProps = { required: boolean };
+
+type CheckboxStoryProps = {
+  helpId: string;
+  labelClassName: string;
+} & FormStoryProps;
+
+export const _Default: StoryObj<typeof CheckboxGroup> = {
+  render: ({ required, helpId, labelClassName }) => (
+    <FormResults
+      onSubmit={() => {
+        console.log('submitted');
+      }}
+      initialValues={{
+        checkboxGroup: [],
+      }}
+      validationSchema={yup.object().shape({
+        checkboxGroup: yup.array().isRequired(required, 'At least one checkbox is required'),
```

Comment:
it looks like there's an alternative way to code the `isRequired` bit.

`validationSchema={yup.object().shape({
        checkboxGroup: yup.array().when([], {
          is: () => required,
          then: yup.array().required("At least one checkbox is required"),
          otherwise: yup.array().notRequired(),
        }),
      })}
`

so in general the schema would be

`{field name}: yup.{string/array/object}.when([], {
          is: () => required,
          then: yup.{string/array/object}().required("This field is required"),
          otherwise: yup.{string/array/object}().notRequired(),
        }),
      })}
`